src/main/java/me/Danker/features/GolemSpawningAlert.java

```diff
@@ -46,12 +47,12 @@ public void renderPlayerInfo(RenderOverlayEvent event) {
 
             double timeNow = System.currentTimeMillis() / 1000;
             mc.getTextureManager().bindTexture(GOLEM_ICON);
-            Gui.drawModalRectWithCustomSizedTexture(MoveCommand.golemTimerXY[0], MoveCommand.golemTimerXY[1], 0, 0, 16, 16, 16, 16);
-
-            String golemText = GOLEM_COLOUR + Utils.getTimeBetween(timeNow, golemTime);
-            new TextRenderer(mc, golemText, MoveCommand.golemTimerXY[0] + 20, MoveCommand.golemTimerXY[1] + 5, 1);
 
             GL11.glScaled(scaleReset, scaleReset, scaleReset);
+            RenderUtils.drawModalRectWithCustomSizedTexture(MoveCommand.golemTimerXY[0] / scale, MoveCommand.golemTimerXY[1] / scale, 0, 0, 16, 16, 16, 16);
+
+            String golemText = GOLEM_COLOUR + Utils.getTimeBetween(timeNow, golemTime);
+            new TextRenderer(mc, golemText, MoveCommand.golemTimerXY[0] + 20 * scale, MoveCommand.golemTimerXY[1] + 5 * scale, scale);
```

Comment:
Golem icon is in an incorrect position when scaled

Reply:
Resolved in commit https://github.com/bowser0000/SkyblockMod/pull/130/commits/de9c63d8229278b90adec39eb5894ea6a79a3dc7